daemon/api.go

```diff
@@ -75,6 +75,9 @@ var api = []*Command{
 	createUserCmd,
 	buyCmd,
 	paymentMethodsCmd,
+}
+
+var publicAPI = []*Command{
 	snapctlCmd,
```

Comment:
Yeah I'm sure there are others as well, I didn't feel super qualified to make that call... perhaps another PR?